server/config.js

```diff
@@ -40,6 +40,7 @@ module.exports = {
     password: get('DB_PASS', 'form-builder'),
     server: get('DB_SERVER', 'localhost', true),
     database: get('DB_NAME', 'form-builder', true),
+    port: get('DB_PORT', 5432, true),
```

Comment:
`DB_PORT` is new and the biggest risk here. 

`5432` is the default port for postgres - but do all environments use `5432` ? 

My best guess is this is fine, but I'm not absolutely certain.